Kotlin-compose/build/compose-benchmarks-benchmarks.uninstantiated.mjs

```diff
@@ -0,0 +1,443 @@
+        'org.w3c.dom.code_$external_prop_getter' : (_this) => _this.code,
+        'org.w3c.dom.reason_$external_prop_getter' : (_this) => _this.reason,
+        'org.w3c.dom.parsing.DOMParser_$external_fun' : () => new DOMParser(),
+        'org.w3c.dom.parsing.parseFromString_$external_fun' : (_this, p0, p1) => _this.parseFromString(p0, p1),
+        'org.w3c.dom.url.URLSearchParams_$external_fun' : (p0, isDefault0) => new URLSearchParams(isDefault0 ? undefined : p0, ),
+        'org.w3c.dom.url.get_$external_fun' : (_this, p0) => _this.get(p0),
+        'org.w3c.fetch.status_$external_prop_getter' : (_this) => _this.status,
+        'org.w3c.fetch.ok_$external_prop_getter' : (_this) => _this.ok,
+        'org.w3c.fetch.statusText_$external_prop_getter' : (_this) => _this.statusText,
+        'org.w3c.fetch.headers_$external_prop_getter' : (_this) => _this.headers,
+        'org.w3c.fetch.body_$external_prop_getter' : (_this) => _this.body,
+        'org.w3c.fetch.blob_$external_fun' : (_this, ) => _this.blob(),
+        'org.w3c.fetch.get_$external_fun' : (_this, p0) => _this.get(p0),
+        'org.w3c.performance.performance_$external_prop_getter' : (_this) => _this.performance,
+        'org.w3c.performance.now_$external_fun' : (_this, ) => _this.now(),
+        'org.w3c.xhr.XMLHttpRequest_$external_fun' : () => new XMLHttpRequest(),
+        'org.w3c.xhr.status_$external_prop_getter' : (_this) => _this.status,
+        'org.w3c.xhr.open_$external_fun' : (_this, p0, p1, p2, p3, p4, isDefault0, isDefault1) => _this.open(p0, p1, p2, isDefault0 ? undefined : p3, isDefault1 ? undefined : p4, ),
+        'org.w3c.xhr.send_$external_fun' : (_this, ) => _this.send(),
+        'org.w3c.xhr.overrideMimeType_$external_fun' : (_this, p0) => _this.overrideMimeType(p0),
+        'androidx.compose.runtime.internal.weakMap_js_code' : () => (new WeakMap()),
+        'androidx.compose.runtime.internal.set_$external_fun' : (_this, p0, p1) => _this.set(p0, p1),
+        'androidx.compose.runtime.internal.get_$external_fun' : (_this, p0) => _this.get(p0),
+        'org.jetbrains.skiko.w3c.language_$external_prop_getter' : (_this) => _this.language,
+        'org.jetbrains.skiko.w3c.userAgent_$external_prop_getter' : (_this) => _this.userAgent,
+        'org.jetbrains.skiko.w3c.navigator_$external_prop_getter' : (_this) => _this.navigator,
+        'org.jetbrains.skiko.w3c.performance_$external_prop_getter' : (_this) => _this.performance,
+        'org.jetbrains.skiko.w3c.open_$external_fun' : (_this, p0, p1) => _this.open(p0, p1),
+        'org.jetbrains.skiko.w3c.window_$external_object_getInstance' : () => window,
+        'org.jetbrains.skiko.w3c.now_$external_fun' : (_this, ) => _this.now(),
+        'org.jetbrains.skia.impl.FinalizationRegistry_$external_fun' : (p0) => new FinalizationRegistry(p0),
+        'org.jetbrains.skia.impl.register_$external_fun' : (_this, p0, p1) => _this.register(p0, p1),
```

Comment:
`register` is called with two args, but below `unregister` is used. Per MDN, "if [unregisterToken is] not provided, the target cannot be unregistered."

Reply:
Thanks for pointing it out! 
I checked the skiko code. Indeed, `unregister` calls will do nothing.

Fortunatelly, there are no bugs due to this as the clode calls `clean` explicitlly along with `unregister`. Anyway, I added a ticket for us to improve it - https://youtrack.jetbrains.com/issue/SKIKO-1040

Reply:
> Though this call is showing up in the profiler as difference in two JS engines

Could you please explain what kind of difference is it?

Reply:
> > Though this call is showing up in the profiler as difference in two JS engines
> 
> Could you please explain what kind of difference is it?

I'm still analyzing that. But my reduced micro-benchmark points on use of Wasm CG objects with `register` function and interaction with Minor GCs.

Reply:
I see fast 2 solutions right now (requiring no Compose release):

- Edit the function in this PR ```'org.jetbrains.skia.impl.register_$external_fun' : (_this, p0, p1) => _this.register(p0, p1, p0)```

-  Or rebuild the benchmarks from this hotfix branch https://github.com/JetBrains/compose-multiplatform/tree/ok/jetstream3_hotfix 

cc: @danleh

Reply:
Thanks for providing the hotfix branch, @eymar. I modified `build.sh` to checkout that branch for the time being and rebuilt from source, which then results in `_this.register(p0, p1, p0)`, as desired.